### ISSUE TITLE
hideheader param on open page not working

### ISSUE THREAD

**Issue by paragajency** [spipu/html2pdf#162]
I had added hideheader param on page , and it is not working as expected. I wanted to hide header & footer  on page 2 and show the header on other pages.  I tried to 
<page   hideheader ="2"backtop="28mm" backbottom="14mm" backleft="14mm" backright="14mm"style="font-size: 12pt" >
    <page_header>
        somcustom header text
    </page_header>
    <page_footer>
        some custom footer text
    </page_footer>
</page

**Comment by fagundes** [comment]
Are you running this on PHP 7? I could find a problem there, it's used split function and it was removed on PHP 7. I'll do a PR to fix this.

**Comment by paragajency** [comment]
hi @fagundes , Thanks for your reply.
No I am on php 5.6
Got it resolved. The problem was space  added between hideheader and assignment operator(=)(<page hideheader ="2").

I had to make it <page hideheader="2" and then it worked.

Also it would have been nice, if hidefooter option was also provided. For now I just used code of hideheader  with changes done for hidefooter, & got hidefooter working

Many thanks
